app/models/propertyDetailsModels.scala

```diff
@@ -82,25 +82,7 @@ object PropertyDetailsValue {
       (JsPath \ "hasValueChanged").readNullable[Boolean]
     )(PropertyDetailsValue.apply _)
 
-  implicit val propertyDetailsValueWrites: OWrites[PropertyDetailsValue] = (
-    (JsPath \ "anAcquisition").writeNullable[Boolean] and
-      (JsPath \ "isPropertyRevalued").writeNullable[Boolean] and
-      (JsPath \ "revaluedValue").writeNullable[BigDecimal] and
-      (JsPath \ "revaluedDate").writeNullable[LocalDate] and
-      (JsPath \ "partAcqDispDate").writeNullable[LocalDate] and
-      (JsPath \ "isOwnedBeforePolicyYear").writeNullable[Boolean] and
-      (JsPath \ "ownedBeforePolicyYearValue").writeNullable[BigDecimal] and
-      (JsPath \ "isNewBuild").writeNullable[Boolean] and
-      (JsPath \ "newBuildValue").writeNullable[BigDecimal] and
-      (JsPath \ "newBuildDate").writeNullable[LocalDate] and
-      (JsPath \ "localAuthRegDate").writeNullable[LocalDate] and
-      (JsPath \ "notNewBuildValue").writeNullable[BigDecimal] and
-      (JsPath \ "notNewBuildDate").writeNullable[LocalDate] and
-      (JsPath \ "isValuedByAgent").writeNullable[Boolean] and
-      (JsPath \ "hasValueChanged").writeNullable[Boolean]
-    )(unlift(PropertyDetailsValue.unapply _))
-
-  implicit val formats = OFormat(propertyDetailsValueReads, propertyDetailsValueWrites)
```

Comment:
if you shift this below there shouldnt be a need to remove it

Reply:
doesnt seem to need a formats, both are implicit in the companion object so should be fine